docs/internals/data-structures.rst

```diff
@@ -961,18 +971,24 @@ key file, wrapped using the standard ``textwrap`` module with a header.
 The header is a single line with a MAGIC string, a space and a hexadecimal
 representation of the repository id.
 
+.. _data-compression:
+
 Compression
 -----------
 
 Borg supports the following compression methods:
 
-- none (no compression, pass through data 1:1)
-- lz4 (low compression, but super fast)
+- none (no compression, pass through data 1:1), identified by ``\x00\x00``
+  (two NUL bytes)
```

Comment:
maybe leave away the "(two NUL bytes)", it doesn't add useful information.

Reply:
My reason for having that was to make it fully unambiguous what is meant by `\x00`. In such specifications/documents, I do prefer to be more precise by stating things in two different ways than leaving something ambiguous.

But I can change it if you disagree despite the rationale :).

Reply:
well, there is still the inconsistency that you explain it only for the none compression, but not for the others.

for the others, if you'ld stay using ascii control char abbreviations, would you like to use SOH as explanation for \x01 and STX for \x02?

i really don't think this is helpful. it "explains" something easy / familiar by using something more complicated / unfamiliar.

that's why i moved the info that there are 2 type bytes to the header (which applies to all) and suggested to drop the ascii char stuff from the none description.

Reply:
Yup, fixed that in the most recent update :-).

(I thought I had fixed it in the update before already, but I somehow messed that up)